### DISCUSSION TITLE
How to turn this into a widget for a website translation

### DISCUSSION THREAD

**Discussion by Immortality-IMT** [Tatoeba/tatoeba2#3135] [Q&A]
How to utilize the work for local website translation, similar widget to https://gtranslate.io/

Cheers

**Comment by Yorwba** [comment]
The easiest way is probably to use the Project Bergamot models that now power [Firefox translation](https://github.com/mozilla/firefox-translations-models). You'll have to train your own models (on Tatoeba data or whatever else you can find) for languages that they don't support out of the box. Expect quality to be slightly worse than what you can get from Google or Microsoft.
